### DISCUSSION TITLE
computer bug recovery

### DISCUSSION THREAD

**Discussion by NEMS6** [community/community#127993] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

bonjour je veux réutiliser mon PC fixe mais il me met cela à chaque fois que je l'allume et puis s'éteint
J'ai essayé de le passer à Ubuntu mais il me dit qu'il ne peut pas le faire car il n'y a pas de connexion et pourtant le câble est branché.

Je voulais savoir si ce serait possible de le réutiliser par tout les moyens possible et inimaginable.

voici les images en pièce jointes si cela pourrai vous aider
Vous pouvez me demander plus de détail si vous en avez besoin et si c'est des détail personnels voici mon adresse mail:

> _**travakali@outlook.fr**_

![pc hakim](https://github.com/community/community/assets/147520411/ff93b451-7aac-4480-919b-ee690ed22de8)
![recuperation pc hakim](https://github.com/community/community/assets/147520411/de8ea4e2-8dda-41eb-862a-aea951c795e7)
### image de quand on l'allume.

**Comment by ebndev** [comment, marked as answer]
Hi @NEMS6, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
